pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -65,8 +65,9 @@ def __call__(self, value, system):
 
         self._request = self.get_request(system)
         # If language present in request, use that. Otherwise, keep language from base class
-        if 'lang' in self._request.GET:
-            self._language = self._request.GET.get('lang')
+        if 'lang' in self._request.GET or 'LANG' in self._request.GET:
```

Comment:
Please use same logic as in pyramid_oereb/views/webservice,py line 751
language = str(self._params.get('LANG')).lower()

Maybe we should even isolate that logic to a function that can be used in both places, webservice and print_proxy?

Reply:
I think it is more a problem of esacping the case of the parameter in the request. [This solution](https://stackoverflow.com/questions/764235/dictionary-to-lowercase-in-python) might work, you must check all the keys to lowercase : `dict((k.lower(), v) for k,v in {'My Key':'My Value'}.iteritems())`